st2common/st2common/models/api/pack.py

```diff
@@ -100,6 +100,22 @@ class PackAPI(BaseAPI):
                                '">=1.8.0, <2.2.0"',
                 'pattern': ST2_VERSION_REGEX,
             },
+            'python_versions': {
+                'type': 'array',
+                'description': ('Major Python versiosn supported by this pack. E.g. '
+                                '"2" for Python 2.7.x and "3" for Python 3.6.x'),
+                'items': {
+                    'type': 'string',
+                    'enum': [
+                        '2',
+                        '3'
+                    ]
+                },
+                'minItems': 1,
```

Comment:
Without a default and minItems = 1 will this break all existing packs?

Reply:
Nope - min items only applies to this attribute when it's provided.

We also have various test cases which cover that scenarios (we have a lot of pack test fixtures without that attribute).